lib/Logger.js

```diff
@@ -0,0 +1,86 @@
+let LogLevel = 0;
+
+const LogLevels = {
+    trace: -2,
+    debug: -1,
+    info: 0,
+    warn: 1,
+    error: 2
+};
+
+const BuildPrefix = (LogLevel) => {
+    let timestamp = new Date().toISOString();
+    return "[" + timestamp + "] [" + LogLevel + "] ";
+};
+
+class Logger {
+    /**
+     * @property {"trace"|"debug"|"info"|"warn"|"error"}
+     */
+    static get LogLevel() {
+        return Object.keys(LogLevels).find(key => LogLevels[key] === LogLevel);
+    }
+    static set LogLevel(value) {
+        if(!LogLevels[value]) {
+            throw "invalid log level '" + value + "'";
```

Comment:
This should also output which levels are valid. To make the error easier to fix for users themselves.